modules/setting-up-posix-shared-memory.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="setting-up-posix-shared-memory_{context}"]
+= Setting up POSIX Shared Memory
+
+
+The following procedure shows how to set up POSIX shared memory between two containers.
+
+.Procedure
+
+1. Create the pod YAML file and specify the tmpfs volume for each container, as shown in the `shared-memory.yaml` file.
++
+====
+.shared-memory.yaml
+[source,yaml]
+----
+---
+apiVersion: v1
+id: hello-openshift
+kind: Pod
+metadata:
+  name: hello-openshift
+  labels:
+    name: hello-openshift
+spec:
+  volumes:                          <1>
+    - name: dshm
+      emptyDir:
+        medium: Memory
+  containers:
+    - image: kubernetes/pause
```

Comment:
This image doesn't actually use /dev/shm. Can we replace it with an example where the containers uses /dev/shm for IPC?